CONTRIBUTING.md

```diff
@@ -27,8 +27,15 @@ Fork using GitHub Desktop:
 - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
 
 Fork using the command line:
+
 - [Fork the repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository) so that you can make your changes without affecting the original project until you're ready to merge them.
 
+Fork, edit and preview using [GitHub Codespaces](https://github.com/features/codespaces):
```

Comment:
Can we link to the Codespaces docs here (similar to the command line instructions? 

Also, we can explain a little bit about why it's nice to use Codespaces... like this: 

```suggestion
[Fork, edit and preview](https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/creating-a-codespace) using [GitHub Codespaces](https://github.com/features/codespaces) to make changes without having to install and run the project locally.
```